community/zh-cn/team.md

```diff
@@ -0,0 +1,66 @@
+# CarbonData 团队
+
+CarbonData 团队由成员和贡献者组成。 成员可以直接访问 CarbonData 项目的源代码并积极地开发代码库。 贡献者通过向成员提交补丁和建议来改善项目。 该项目的贡献者数量是无限的。 无论是琐碎的清理工作，重要的新功能还是其他重大的奖励，对 CarbonData 所做的所有贡献都将受到极大的赞赏。
```

Comment:
CarbonData's PMC and Committer info at :  https://cwiki.apache.org/confluence/display/CARBONDATA/PMC+and+Committers+member+list

Reply:
ok